checker/src/main/java/org/checkerframework/checker/nullness/KeyForAnalysis.java

```diff
@@ -27,6 +27,11 @@ public KeyForStore createEmptyStore(boolean sequentialSemantics) {
         return new KeyForStore(this, sequentialSemantics);
     }
 
+    @Override
+    public KeyForStore createBottomStore(boolean sequentialSemantics) {
+        return new KeyForStore(this, sequentialSemantics);
```

Comment:
Maybe just `return createEmptyStore(sequentialSemantics)`?

Reply:
`return new KeyForStore(this, sequentialSemantics, false);`